packages/api/src/hooks/p2p/entity/advert/p2p-advert/useAdvertList.ts

```diff
@@ -14,7 +14,7 @@ const useAdvertList = (
         payload: { ...payload, offset: payload?.offset, limit: payload?.limit },
         options: {
             getNextPageParam: (lastPage, pages) => {
-                if (!lastPage?.p2p_advert_list?.list) return;
+                if (lastPage?.p2p_advert_list?.list.length === 0 || !lastPage?.p2p_advert_list?.list) return;
```

Comment:
can we not club it into !lastPage?.p2p_advert_list?.list?.length?

Reply:
itll always return an empty list if there is not next page in this case, so its better we check if its 0